packages/ProgressIndicators/src/index.tsx

```diff
@@ -1 +1,2 @@
-export * from './ProgressBar';
```

Comment:
What was wrong with having it this way?

Reply:
I don't remember the actual reason we changed it  but I can always have it this way and test nothing is broken

Reply:
I get this error when I import progress bar in reveal `JSX element type 'ProgressBar' does not have any construct or call signatures.`

That is the main reason we added that.